tests/manage/z_cluster/cluster_expansion/test_node_expansion.py

```diff
@@ -36,4 +37,6 @@ def test_add_node_aws(self):
             else:
                 node_type = 'RHCOS'
             assert add_new_node_and_label_upi(node_type, new_nodes), "Add node failed"
+            if config.ENV_DATA.get('rhel_workers'):
+                assert post_ocp_workaround(), "Adding workaround for the issue #1151 failed"
```

Comment:
IMO it would be better to add this workaround directly in `add_new_node_and_label_upi()` so that we don't need to apply this workaround in every test.
Besides, it will become easier to remove this WA later.

Reply:
Ack